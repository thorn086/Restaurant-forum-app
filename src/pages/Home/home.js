import React from 'react'
import Title from '../../components/NavBar/NavBar'
import NavBar from '../../components/Title/Title'

function Home() {
    return (
    <div>
        <header className='header'>
        <Title />
        <br />
        <NavBar />
        </header>
    </div>
    )
}
export default Home