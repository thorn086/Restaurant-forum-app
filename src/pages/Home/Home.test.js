import React from 'react';
import Home from './Home';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe(' render properly', () => {
    const home = shallow(<Home/>);
    it('renders properly', () => {
        expect(home).toMatchSnapshot();
    });
    it('contains a NavBar component', () => {
        expect(home.find('NavBar').exists()).toBe(true);
    });

})