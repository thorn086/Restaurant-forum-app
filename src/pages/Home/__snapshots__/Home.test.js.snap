// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` render properly renders properly 1`] = `ShallowWrapper {}`;
